spec/beaker/hypervisor/aws_sdk_spec.rb

```diff
@@ -672,60 +679,14 @@ module Beaker
     describe '#ensure_key_pair' do
       let( :region ) { double('region', :name => 'test_region_name') }
       subject(:ensure_key_pair) { aws.ensure_key_pair(region) }
+      let( :key_name ) { "Beaker-rspec-SUT" }
 
-      context 'when a beaker keypair already exists' do
```

Comment:
they were actually bad tests that don't give us any meaningful results.  We haven't used cached keys and the `:keypair_generate_new` option hasn't generated new keys since BKR-522.  This should have been pulled there, but I missed them then.